examples/ex-batched-matrix-multiply.cpp

```diff
@@ -128,9 +129,9 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 // in an array object. Here double braces are used to initialize the array and its
 // subobjects (number of entries in each component).
 // The layout generates an indexing equivalent to
-// A(e,r,c) A[c + NCOLS*(r + NROWS*e)]
+// A(e,r,c) A[c + N_c*(r + N_r*e)]
   auto layout =
-      RAJA::make_permuted_layout({{N, NROWS, NCOLS}},
+      RAJA::make_permuted_layout({{N, N_r, N_c}},
```

Comment:
It would be helpful to say that the column index has unit stride in the comment below; e.g., 'the column index has unit stride; i.e., arg 2'

Reply:
Thanks, I will address that. In the example above I tried to create an analogue using the layout and creating a macro with the desired indexing. I was wondering if it is clear? or if there may be a better way to describe it.

Reply:
I think it's fine with some minor additions to the comments that I noted.